app/views/buyers/applications/_listing.html.slim

```diff
@@ -28,7 +28,7 @@ table class="data"
       th = sortable('cinstances.state', 'State')
       - unless @account
         th = sortable('accounts.org_name', 'Account')
-      - if current_account.multiservice?
+      - if @show_services
```

Comment:
I do not really like using this kind of variable in a template

Reply:
@hallelujah Ok, what do you propose then? I have to show it when it is one view and not with the other, so the other possibility that comes to mind is show it or not with JS depending on the current URL, which sounds even worse to me. So what do you propose? 😝